reactnative/ReactNativeTemplateApp/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'android'
 
 dependencies {
-    compile project(':libs:SmartSync')
+    compile project(':libs:SalesforceSDK')
```

Comment:
Oh right - now I remember. We generate everything with ```:libs:SalesforceSDK``` and replace with ```:libs:SmartSync``` if required. We could do the same thing as ```native``` apps by offering the option, or we could do what we do on ```hybrid```, which is including ```SmartSync``` by default. Given the current trend in questions on the community, it looks like people are using ```SmartSync``` quite a bit, so maybe we should bundle by default? I'm okay either way, since we have both paths already coded :-)